### ISSUE TITLE
"Access denied" error for /webpack-dev-server/index.html in Edge

### ISSUE THREAD

**Issue by fcntl** [webpack/webpack-dev-server#639]
webpack-dev-server 1.16.1
webpack 1.13.2

Works fine in Chrome. In Edge however, I get:

"Access denied"

or

"WebSocket Error: Network Error 12152, The server returned an invalid or unrecognized response"

I was following this guide to get live refresh working:
https://webpack.github.io/docs/webpack-dev-server.html#iframe-mode

**Comment by SpaceK33z** [comment]
Where exactly do you see "Access denied"? Can you show a screenshot (also with URL)?

**Comment by fcntl** [comment]
@SpaceK33z well I saw that previously, now it is just saying "http://localhost:3000/webpack-dev-server/index.html" in the console. All I know is it loads in Chrome, doesn't load in Edge, side-by-side.

http://imgur.com/a/0EzCA

**Comment by SpaceK33z** [comment]
Interesting. I turned on my Windows machine to test this, and while I don't get that WebSocket error you're having, the contents of the iframe don't load.

As you said, this works in Chrome. It also even works in IE12... I noticed that in Edge the iframe `src` is `javascript:;`, while it should be `/` (so where your app lives), but have no idea why.

If anyone wants to make a PR for this with a fix, that is welcome. I am not going to waste my time for this Edge-specific bug. I thought Edge would make our lives as a web developer easier, not harder :(.
